bin/i18n/hourofcode_crowdin.yml

```diff
@@ -2,25 +2,23 @@
 # Your crowdin's credentials
 #
 "project_identifier" : "hour-of-code"
-"base_path" : "../../i18n/locales/source/hourofcode"
+"base_path" : "../../i18n/locales"
 
 # API Credentials must be loaded from a separate identity file. See
 # https://support.crowdin.com/configuration-file/#split-project-configuration-and-api-credentials
 "api_key" : ""
 
-"preserve_hierarchy": true
```

Comment:
I believe we want to preserve this; otherwise, source files will all be uploaded to the root directory of the crowdin project

Reply:
They were being uploaded into the `source` directory of the crowdin project with it, which was extra strange

Reply:
The dryrun above was without this flag and it looks right? The `code-dot-org` crowdin project doesn't have this setting so I'm not sure why this one is different?

Reply:
Okay I read the [documentation](https://support.crowdin.com/configuration-file/#saving-directory-structure-on-serve and it isn't clear. If `preserve_hierarchy` is set to true it will preserve the whole hierarchy but I'm not sure what happens when it's _not_ set.

Reply:
upon further investigation, it looks like we actually aren't using it in the config for the regular codeorg sync either, so I think I had some misunderstanding about how this is used